.github/workflows/e2e.yml

```diff
@@ -0,0 +1,124 @@
+name: e2e
+
+on:
+  push:
+    branches: [ master ]
+env:
+  REGISTRY: ${{ secrets.REGISTRY}}
```

Comment:
It's not need to create the secret for registry. You can set temporary repo, because we don't push images anywhere. For example: `ghcr.io/dell/csi-baremetal/`. But, please, add comment too